From 4cb4df057df03ce1ccc81b2778b5090d1443ae15 Mon Sep 17 00:00:00 2001
From: Hannes Reinecke <hare@suse.de>
Date: Fri, 21 Mar 2014 14:46:40 +0100
Subject: systemctl cannot start fcoemon.socket

systemctl start fcoemon.socket fails, as systemd cannot find
the referenced service.

References: bnc#869392

Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 etc/systemd/fcoemon.socket | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/etc/systemd/fcoemon.socket b/etc/systemd/fcoemon.socket
index 4de8715..fdf7141 100644
--- a/etc/systemd/fcoemon.socket
+++ b/etc/systemd/fcoemon.socket
@@ -1,4 +1,8 @@
+[Unit]
+Description=Open-FCoE daemon control socket
+
 [Socket]
+Service=fcoe.service
 ListenDatagram=@/com/intel/fcoemon
 PassCredentials=true
 
-- 
1.8.4.5

